### DISCUSSION TITLE
Prevent highlighted text to become accidental cloze cards

### DISCUSSION THREAD

**Discussion by mtomas7** [st3v3nmw/obsidian-spaced-repetition#244] [General]
Thank you very much for your plugin!

I have such a problem: usually, in one document I write my notes on the subject and at the end of the file I include test questions. If I use ==highlights== in the notes, plugin picks them up as cloze cards. I guess, I could separate questions to a separate file, but that that would make note structure unnecessary complicated.

I was thinking if it would be possible to use HTML comments to separate text that should be excluded from the scanning for questions, eg:

`<!-- Do Not Scan for Flash cards - Start -->`

and close it with

`<!-- Do Not Scan for Flash cards - Stop -->`

Or something similar. Or otherwise, we could use comment to show the place of questions:

`<!-- Scan for Flash cards - Start -->`

close:

`<!-- Scan for Flash cards - Stop -->`

How do you think?

Thank you!

**Comment by DryIce1** [comment]
Or perhaps using the {{Curly Brackets}} would be easier?
- Personally, I would prefer to only use ==highlights== in my notes for highlights. 

Thanks!

**Reply by cesar-programs** [reply]
Yes, custom cloze syntax would be nice.

A workaround could be to use the standard markdown bold syntax, `**bold**`, for clozes that we want to make into flash cards and then we could use the alternative bold syntax, `__alt bold__`, for when we want to bold but not make a flash card

Also, checkout the community plug-in 'Extra Markdown Commands' to make a hotkey for the alternative syntax (non flash card bold).

Then remap standard bold syntax (cloze bold) syntax to something like Anki, for example: CMD+Shift+C 

<img width="733" alt="Screen Shot 2021-12-15 at 19 22 15" src="https://user-images.githubusercontent.com/84650530/146302701-6d3c1281-39fd-4da9-b8bd-b8ce2b2a164a.png">

**Comment by MotaOcimar** [comment]
Related to #387
